devicehive-shim-api/src/main/java/com/devicehive/shim/api/Action.java

```diff
@@ -71,6 +71,13 @@
     LIST_DEVICE_RESPONSE,
     COUNT_DEVICE_REQUEST,
 
+    LIST_PLUGIN_REQUEST,
+    LIST_PLUGIN_RESPONSE,
+    COUNT_PLUGIN_REQUEST,
+
+    LIST_SUBSCRIBE_REQUEST,
```

Comment:
It was deleted in previous PR. Probably, wrong merge conflict strategy has been used.